src/tls/openssl/tls.c

```diff
@@ -2153,6 +2154,38 @@ int tls_set_resumption(struct tls *tls, const enum tls_resume_mode mode)
 }
 
 
+/**
+ * Set TLS OCSP stapling mode
+ *
+ * @param tls  TLS Object
+ * @param mode TLS OCSP stapling mode
+ *
+ * @return 0 if success, otherwise errorcode
+ */
+int tls_set_ocsp_stapling(struct tls *tls, const enum tls_ocsp_stapling mode)
```

Comment:
have this been tested with LibreSSL ?

Reply:
At least through the `retest`, yes. Initially, I had used some OpenSSL functions which were not supported by LibreSSL, and the `retest` showed this, so I refactored it. So I'm pretty sure that this also works with LibreSSL.